.travis.yml

```diff
@@ -0,0 +1,35 @@
+language: perl
+perl:
+  - "5.30"
+  - "5.28"
+  - "5.26"
+  - "5.24"
+  - "5.22"
+  - "5.20"
+  - "5.18"
+  - "5.16"
```

Comment:
Might be worth adding a comment as to why the tests begin at 5.16.  Is there a reason they can't begin with earlier versions?